application/values.yaml

```
@@ -73,6 +73,7 @@ autoscaling:
   minReplicas: 1
   maxReplicas: 100
   targetCPUUtilizationPercentage: 80
+  enable_rabbitmq_custom_metric: false
```

to make this generic we could define the values.yaml in a different way here:
```suggestion
  customMetrics:
    metricName:
      describedObject:
        ...
      target:
        ...
```